### ISSUE TITLE
About the AR model

### ISSUE THREAD

**Issue by bestbzw** [qizhex/RACE_AR_baselines#2]
Hi
I re-build a AR model with tensorflow,  but the loss not decrease. i can't find the difference  between your model and mine. could you  public the training  log?
Thank you very much!

**Comment by qizhex** [comment]
Sorry for the late response. The loss plateaus in the middle phase of the training but will gradually improve at the later stage. Did you train the model for 100 epochs?
